target/linux/ipq40xx/image/Makefile

```diff
@@ -32,6 +32,19 @@ define Device/UbiFit
 endef
 
 
+define Device/8dev_jalapeno
+	$(call Device/FitImage)
+	$(call Device/UbiFit)
+	DEVICE_DTS := qcom-ipq4018-jalapeno
+	KERNEL_INSTALL := 1
+	KERNEL_SIZE := 4048k
```

Comment:
Where does the kernel size limit comes from? It looks like the kernel is a ubi volume. In that case there is usually not size restriction for the kernel.

Reply:
Same as above.
Yes,everything is on ubi so there is no actuall limit